generatesitemap.py

```diff
@@ -220,8 +220,12 @@ def lastmod(f) :
     mod = subprocess.run(['git', 'log', '-1', '--format=%cI', f],
                     stdout=subprocess.PIPE,
                     universal_newlines=True).stdout.strip()
+    print(date_only) # trying to debug what's going wrong...
     if len(mod) == 0 :
         mod = datetime.now().astimezone().replace(microsecond=0).isoformat()
+    if date_only == "true":
+        date_only = '%Y-%m-%d'
+        mod = datetime.strptime(mod, '%Y-%m-%dT%H:%M:%S%z').strftime(date_only)	
```

Comment:
This is more complicated than it needs to be. At this point, before reformatting, `mod` is guaranteed to be formatted as a complete date and time. Just use a Python slice here to keep the date portion of that.

Reply:
This is my problem I was struggling with. Can I get a clear example of the Boolean.. 

The action is set default to "false" which is a Boolean value. Tho when using a simple if statement: and nothing else.. things DO not work proper.... 

I would more than appreciate a demo with this. As I can not figure out what I am doing wrong.